BUILDING.md

```diff
@@ -27,15 +27,15 @@ guide, before continuing further.
 The project depends on [CMake](https://cmake.org/) >= 3.25,
 [Boost](https://www.boost.org/) >= 1.74, [gRPC](https://grpc.io/) >=
 1.30.2, [protobuf](https://protobuf.dev/overview/) >= 3.12.4,
-[xtensor](https://github.com/xtensor-stack/xtensor) >= 0.24.3,
-[abseil](https://abseil.io/) and on the transitive dependencies of
+[xtensor](https://github.com/xtensor-stack/xtensor) >= 0.24.3
+and on the transitive dependencies of
 those projects.
 
 You will need to install these required dependencies before building
 the Viam C++ SDK, preferably by way of your system package manager:
 
-- Debian: `apt-get install cmake build-essential libabsl-dev libboost-all-dev libgrpc++-dev libprotobuf-dev libxtensor-dev`
-- MacOS with Homebrew: `brew install abseil cmake boost grpc protobuf xtensor`
+- Debian: `apt-get install cmake build-essential libboost-all-dev libgrpc++-dev libprotobuf-dev libxtensor-dev`
```

Comment:
I think there are some references in the top level CMakeLists about absl as well.